docs/src/concepts.md

```diff
@@ -0,0 +1,40 @@
+# Concepts
+
+## Modeling the Brain
+
+Studies of brain anatomy and cellular morphology were once confined to single-subject studies wherein in vitro slices were compared to develop a complete model of the brain. With dyes, researchers could compare recordings to the reconstructed location of a given recording device. Through clever experimental design, researchers could further associate highly localized regions to their task-dependent function. Advances in 3D modeling have permitted parallel advances in multi-subject averaged anatomical models. An atlas serves as a shared reference frame for a given species. [The Allen Institute](https://mouse.brain-map.org/) has been a leader in the development of mouse atlases since 2007<sup>[1](#references)</sup>, with regular updates<sup>[2](#references)</sup> that provide researchers with a shared reference frame in the study of functional neuroanatomy.
+
+## Element Architecture
+
+Each of the DataJoint Elements are a set of tables for common neuroinformatics modalities to organize, preprocess, and analyze data. Each node in the following diagram is either a table in the Element itself or a table that would be connected to the Element.
+
+![element-electrode-localization diagram](https://raw.githubusercontent.com/datajoint/element-electrode-localization/main/images/diagram_electrode_localization.svg)
+
+The Element is separated into two schemas:
+
++ `coordinate_framework` ingests standard atlases and retains voxel-based lookup tables, including references to brain region information, acronyms and standardized color codes.
++ `electrode_localization` pairs the above reference tables with Neuropixels probe location data in a `probe` schema, optionally from [Element Array Ephys](https://github.com/datajoint/element-array-ephys)
+
+## Key Partnerships
+
+In coordination with both the Mesoscale Activity Project and the International Brain Lab, DataJoint developed mechanisms for pairing recording coordinates with the published atlases in project-specific cases that have since been generalized.
+
+## Pipeline Development and Limitations
+
+The Element's table architecture was generalized from existing project-specific
+pipelines such as International Brain Laboratory's `channel_locations.json` as well as the Allen
+Institute's [atlas
+files](https://community.brain-map.org/t/allen-mouse-ccf-accessing-and-using-related-data-and-tools/359),
```

Comment:
careful of linebreaks in links

Reply:
It works in my local deployment

Reply:
my recollection is that this breaks when deployed on gh-pages, but that was me working on my personal site, without the dockerization. We can make a note to confirm after merged or I can pull to my fork to launch gh-pages

Reply:
I can confirm after the merge if that's easier

Reply:
This works on the live site